util/UnicodeTable.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include <codecvt>
+#include "UnicodeTable.h"
+
+void UnicodeTable::printLine(std::string rowName, int length) {
```

Comment:
This is the base for a unicode table for debugging, I'll finish it in another branch before next meet up.

Reply:
This is gonna be nice